### PR TITLE
Translation update (Danish)

### PR DESCRIPTION
**Pull request by matsduf**
PR #821 was based on master branch instead of develop branch and created conflicts when created against develop branch. This PR is meant to be a replacement of #821.

* Commit 5acd6a5 is created by cherry-picking the commit 58ae64f from #821 and accepting everything from that.
* Commit d1a36b3 remove some duplicate or misplaced entries in the resulting `da.po`.
* Commit 03a9d53 adjusts all lines (msgstr) to the standard line lenghts (no change of content).

### CODE REVIEW

**Comment by matsduf** [comment]
The entire file can be inspected by https://github.com/zonemaster/zonemaster-engine/blob/03a9d5314c28ea5c0ba307e3354a100582c5dc4f/share/da.po

To download the entire file, use https://raw.githubusercontent.com/zonemaster/zonemaster-engine/03a9d5314c28ea5c0ba307e3354a100582c5dc4f/share/da.po

File: share/da.po [hunk 1]
```diff
@@ -1137,11 +1185,11 @@ msgstr ""
...
 #, perl-brace-format
 msgid ""
-"Parent does not list enough ({count}) nameservers ({glue}). Lower limit set "
-"to {minimum}."
+"Parent does not list enough ({count}) nameservers ({nsname_list}). Lower "
+"limit set to {minimum}."
 msgstr ""
-"Forælderen returnerer ikke tilstrækkeligt antal ({count}) navneservere "
-"({glue}). Mindsteværdi er sat til {minimum}."
+"Barnet indeholder ikke tilstrækkeligt antal ({count}) navneservere "
```
Comment (mattias-p) [inline share/da.po:1191]: "Barnet" doesn't seem right here.

Comment (matsduf) [reply share/da.po:1191]:
```
#. DELEGATION:NOT_ENOUGH_IPV4_NS_CHILD
#, perl-brace-format
msgid ""
"Child does not list enough ({count}) nameservers ({nsname_list}) that "
"resolve to IPv4 addresses ({ns_ip_list}). Lower limit set to {minimum}."
msgstr ""
"Barnet indeholder ikke tilstrækkeligt antal ({count}) navneservere "
"({nsname_list}) med IPv4-adresser ({ns_ip_list}). Minimumsværdien er "
"{minimum}."
```

What is the issue with "barnet"?

**Comment by matsduf** [comment]
@mattias-p, please re-review.

File: share/da.po [hunk 2]
```diff
@@ -498,13 +527,28 @@ msgstr ""
...
+"{ns} failed to be verified with error '{error}' (a DS record with same tag "
+"is present in the parent zone)."
+msgstr ""
+"RRSIG bag DNSKEY RR-sæt oprettet af tag {keytag} returneret af navneserveren "
+"{ns} fejlede verifikation med fejlen '{error}' (en DS record med samme tag "
+"er til stede i forældrezonen)."
 
 #. DNSSEC:DELEGATION_NOT_SIGNED
-#, perl-brace-format
+#, fuzzy, perl-brace-format
```
Comment (mattias-p) [inline share/da.po:551]: I found a fuzzy mark!

File: share/da.po [hunk 3]
```diff
@@ -1060,11 +1106,11 @@ msgstr ""
...
 #, perl-brace-format
 msgid ""
-"Parent lists enough ({count}) nameservers ({glue}). Lower limit set to "
-"{minimum}."
+"Parent lists enough ({count}) nameservers ({nsname_list}). Lower limit set "
+"to {minimum}."
 msgstr ""
-"Forælderen returnerer tilstrækkeligt antal ({count}) navneservere ({glue}). "
-"Mindsteværdi sat til {minimum}."
+"Barnet indeholder tilstrækkeligt antal ({count}) navneservere "
```
Comment (mattias-p) [inline share/da.po:1112]: Wasn't this fixed already? Maybe there were more and I only spotted one of them?

File: share/da.po (hunk 1 above)
Comment (mattias-p) [reply share/da.po:1193]: You're quoting the wrong message. This is DELEGATION:NOT_ENOUGH_NS_DEL. It begins with "Parent".

File: share/da.po (hunk 2 above)
Comment (matsduf) [reply share/da.po:551]: Fixed.

File: share/da.po (hunk 1 above)
Comment (matsduf) [reply share/da.po]: Fixed the one with typo.

**Comment by matsduf** [comment]
@mattias-p, please re-review.

File: share/da.po (hunk 3 above)
Comment (mattias-p) [reply share/da.po:1112]: This one (DELEGATION:NOT_ENOUGH_NS_DEL) still has "Parent" in msgid and "Barnet" in msgstring.

Comment (mattias-p) [reply share/da.po:1112]: This one (DELEGATION:ENOUGH_NS_DEL) still has "Parent" in msgid and "Child" in msgstr.

**Review by sandoche2k — APPROVED** [review]

**Comment by matsduf** [comment]
Corrected and updated. Please re-review, @mattias-p and @sandoche2k.

**Review by sandoche2k — APPROVED** [review]

**Review by mattias-p — APPROVED** [review]

**Pull request merged**